### PR TITLE
[PCFG-170] Massive loading CI-Station

### PR DESCRIPTION
**Pull request by aomegax**
#### List of Changes
- massive loading of creditor institution - station relationship

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/pagopa/apiconfig/service/MassiveLoadingService.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+
+@Service
+@Validated
```
Comment (jacopocarlini) [inline src/main/java/it/pagopa/pagopa/apiconfig/service/MassiveLoadingService.java:32]: use @Valid instead

File: src/main/java/it/pagopa/pagopa/apiconfig/model/massiveloading/CreditorInstitutionStation.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    public enum Env {
+        ESER, COLL;
+    }
+
+    public enum YesNo {
+        S, N;
+    }
+
+    public enum Operation {
+        A, C;
```
Comment (jacopocarlini) [inline src/main/java/it/pagopa/pagopa/apiconfig/model/massiveloading/CreditorInstitutionStation.java:52]: add a comment to explain

**Comment by jacopocarlini** [comment]
there are 3 code smells

File: src/main/java/it/pagopa/pagopa/apiconfig/service/MassiveLoadingService.java (hunk 1 above)
Comment (aomegax) [reply src/main/java/it/pagopa/pagopa/apiconfig/service/MassiveLoadingService.java]: done

File: src/main/java/it/pagopa/pagopa/apiconfig/model/massiveloading/CreditorInstitutionStation.java (hunk 2 above)
Comment (aomegax) [reply src/main/java/it/pagopa/pagopa/apiconfig/model/massiveloading/CreditorInstitutionStation.java:55]: done

**Review by jacopocarlini — APPROVED** [review]
good job

**Pull request merged**